src/js/components/FilterByTaskState.js

```diff
@@ -56,13 +57,22 @@ class FilterByTaskState extends React.Component {
   }
 
   render() {
+    let {inverseStyle} = this.props;
+
+    let dropdownMenuClassSet = classNames({
```

Comment:
i do believe we should move towards having the classes replacable like lunoe is saying though. the amount of times we've run into trouble with this issue is significant. the question of whether it should happen in this PR i'm not really sure.